### PR TITLE
for 2.1, translate all strings to German, which haven't been translated yet

### PR DESCRIPTION
**Pull request by Strubbl**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | yes
| Fixed tickets | comma-separated list of tickets fixed by the PR, if any
| License       | MIT

Könnt ihr bitte meine Übersetzung auf Fehler/Verbesserungen prüfen, @mruminski @jlnostr ?

@nicosomb i translated some more missing strings than the quickstart ones. Hope that is okay.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.de.yml [hunk 1]
```diff
@@ -297,12 +297,12 @@ quickstart:
...
         title: 'Entwickler'
-        # description: 'We also thought to the developers: Docker, API, translations, etc.'
+        description: 'Wir haben auch an die Entwickler gedacht: Docker, API, Übersetzungen, etc.'
         create_application: 'Erstelle eine Anwendung und nutze die wallabag API'
-        # use_docker: 'Use Docker to install wallabag'
+        use_docker: 'Nutze Docker, um wallabag zu installieren'
     docs:
         title: 'Komplette Dokumentation'
-        # description: "There are so much features in wallabag. Don't hesitate to read the manual to know them and to learn how to use them."
+        description: "Es gibt viele Features in wallabag. Scheue dich nicht die Anleitung zu lesen, um sie kennenzulernen und zu lernen sie zu nutzen."
```
Comment (jlnostr) [inline src/Wallabag/CoreBundle/Resources/translations/messages.de.yml]: I'm not 100% happy with the translation of the second sentence. Sounds too complicated for me. Maybe @mruminski has an idea?

**Comment by j0k3r** [comment]
Something is wrong line 34

> Unable to parse at line 34 (near " users_management: 'Benutzerverwaltung'").

File: src/Wallabag/CoreBundle/Resources/translations/messages.de.yml (hunk 1 above)
Comment (Strubbl) [inline src/Wallabag/CoreBundle/Resources/translations/messages.de.yml]: Vielleicht:

Zögere nicht, die Features im Handbuch zu erkunden und zu lernen, wie sie funktionieren.

Comment (jlnostr) [inline src/Wallabag/CoreBundle/Resources/translations/messages.de.yml]: Klingt deutlich besser :+1:

**Pull request merged**